pkg/okteto/context.go

```diff
@@ -221,15 +221,10 @@ func ContextStore() *OktetoContextStore {
 		}
 		CurrentStore = ctxStore
 
-		if _, ok := CurrentStore.Contexts[CurrentStore.CurrentContext]; !ok {
-			return CurrentStore
-		}
 		return CurrentStore
 	}
 
-	CurrentStore = &OktetoContextStore{
-		Contexts: map[string]*OktetoContext{},
-	}
+	CurrentStore = &OktetoContextStore{}
```

Comment:
I think we need to initialise the context map, otherwise it will panic whenever we try to create a new entry to `CurrentStore`.
```suggestion
	CurrentStore = &OktetoContextStore{
	     Contexts: map[string]*OktetoContext{},
	}
```

Reply:
Yep, I found that changing this broke tests. I have a question though. Doesn't omitting the `Contexts` map set it to be zero valued (default value) and I think that that's what's happening in line 228?